canonical-websites/www.ubuntu.com
# Release fixes

## Done

Final changes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Ensure matches changes requested in below issue


## Issue / Card

Fixes #3058
